mountpoint-s3/src/inode.rs

```diff
@@ -80,6 +86,7 @@ pub struct Superblock {
 struct SuperblockInner {
     bucket: String,
     inodes: RwLock<HashMap<InodeNo, Inode>>,
+    known_missing_keys: ExpirableSet,
```

Comment:
Fair question. In fact, it had started that way, but I decided to move the mutex into `ExpirableSet` so that the "inode.negative_cache.operation_duration_us" metrics could also include its contention.
The alternative I see would be to add wrapper methods to `SuperblockInner` and move the locking (and the metrics) there. `ExpirableSet` would still be quite specific though, so I'm not convinced it would be worth it.

Reply:
To clarify, this is not a blocker. Just thinking out loud.

Reply:
Good point. I'll try to rename to something more specific.